codes/classical/q-ary_digits/reed_solomon.yml

```diff
@@ -22,21 +22,39 @@ features:
   decoders:
     - 'Berlekamp-Welch decoder \cite{manual:{E. R. Berlekamp and L. Welch, Error Correction of Algebraic Block Codes. U.S.
 Patent, Number 4,633,470 1986.}}, assuming that \(t \geq (n+k)/2\).'
+    - 'Gao decoder using extended Euclidean algorithm \cite{manual:{S. Gao, "A new algorithm for decoding Reed-Solomon codes." Communications, information and network security. Springer, Boston, MA, 2003. 55-68.}}.'
+    - 'Sudan list decoding algorithm, up to \(1-\sqrt{2R}\) errors \cite{manual:{M. Sudan, “Decoding Reed-Solomon codes beyond the error-correction bound,” Journal of Complexity, vol. 13, no. 1, pp. 180–193, 1997.}}'
```

Comment:
up to \(1-\sqrt{2R}\) _proportion of_  errors; same correction for the Guruswami-Sudan algorithm.